python_modules/dagster/dagster/_core/definitions/declarative_scheduling/operands/slice_conditions.py

```diff
@@ -50,11 +50,36 @@ def compute_slice(self, context: SchedulingContext) -> AssetSlice:
 
 @whitelist_for_serdes
 class InLatestTimeWindowCondition(SliceSchedulingCondition):
-    lookback_seconds: Optional[float] = None
+    lookback_days: int = 0
+    lookback_seconds: int = 0
+    lookback_microseconds: int = 0
+
```

Comment:
Might be worth nothing as comment that

```
>>> timedelta().days
0
>>> timedelta().seconds
0
>>> timedelta().microseconds
0
```
I went to the REPL and checked this to ensure it wasn't `None`.

Reply:
And actually I think you *should* make these optional to distinguish from the zero case.